src/main/java/io/cdap/e2e/pages/locators/CdfGCSLocators.java

```diff
@@ -124,4 +107,19 @@
 
   @FindBy(how = How.XPATH, using = "//textarea[@data-cy='fileSystemProperties']")
   public static WebElement fileSystemProperties;
+
+  @FindBy(how = How.XPATH, using = "//*[@data-cy='fileSystemProperties']//button/span[text()='Tidy']")
+  public static WebElement fileSystemPropertiesTidyButton;
+
+  public static WebElement pathFilenameOnly(String value) {
+    return SeleniumDriver.getDriver().findElement(By.xpath("//input[@name='filenameOnly' and @value='" + value + "']"));
+  }
+
+  public static WebElement recursive(String value) {
+    return SeleniumDriver.getDriver().findElement(By.xpath("//input[@name='recursive' and @value='" + value + "']"));
+  }
+
+  public static WebElement getListItem(String option) {
```

Comment:
Is it a general locator or selecting item from particular list?? From locator name it is not clear. If its generic it should be in some general PluginPropertiesLocators.

Reply:
This currently used only GCSActions but can be used across plugins to get dropdown list item.
I will rename it and add TODO to move it to CdfPluginPropertiesLocators.java as this file is not yet merged.

Reply:
Done.